src/osm_tools/osmconvert.py

```diff
@@ -0,0 +1,46 @@
+import os
+import constants
+
+from uuid import uuid4
+from helper_functions import subprocess_get_stdout_output, run_command_wrapper, deconstruct_file_path
+
+def convert_changefile_format(input_path, input_format, output_format):
+    if input_format is output_format:
+        return
+    (_, input_name, _, _) = deconstruct_file_path(input_path)
+    output_path = os.path.join(constants.OSMCHANGES_PATH, output_format, f'{input_name}.{output_format}')
+    run_command_wrapper(f'{constants.OSMCONVERT_PATH} \
+                    {input_path} \
+                    -o={output_path} \
+                    --verbose')
+
+def get_osm_file_timestamp(file_path):
+    command = [constants.OSMCONVERT_PATH, '--out-timestamp', file_path, '--verbose']
+    try:
+        return subprocess_get_stdout_output(command)
+    except:
+        raise
```

Comment:
it is an empty `try, catch` wrapping, redundant

Reply:
removed.